x-pack/plugins/upgrade_assistant/public/application/components/constants.tsx

```diff
@@ -24,3 +24,5 @@ export const COLOR_MAP: { [level: string]: IconColor } = {
   warning: 'default',
   critical: 'danger',
 };
+
+export const DEPRECATIONS_PER_PAGE = 25;
```

Comment:
Maybe its worth allowing the user to change this number?

Reply:
That's a great point. This was the behavior from the old implementation (for showing ES deprecations). I think I'd prefer to address this in a separate PR.